setup.cfg

```diff
@@ -0,0 +1,8 @@
+[aliases]
+test=pytest
+
+[zest.releaser]
+create-wheel = yes
+
+[bdist_wheel]
+universal=1
```

Comment:
Because Unix tools are looking for it.

Try to do a `cat` on a file with and without the line for instance :)